cmd/utils/flags.go

```diff
@@ -124,7 +124,7 @@ var (
 	}
 	NetworkIdFlag = cli.Uint64Flag{
 		Name:  "networkid",
-		Usage: "Network identifier (integer, 1=Frontier, 2=Morden (disused), 3=Ropsten, 4=Rinkeby)",
+		Usage: "Network identifier (integer, 1=Frontier, 2=Morden (disused), 3=Ropsten, 4=Rinkeby, 5=Ottoman)",
```

Comment:
Seems like we might want to use another (higher) number that's less likely to cause a conflict in the future?

Reply:
It depends on the upstream codebase. We submit the same code to geth, so if that gets merged, we are going to grab network id 5. If we eventually get a larger number, I think we can update it here accordingly.